### PR TITLE
feat(Modal): support placeholder & add theme

### PR DESCRIPTION
**Pull request by paranoidjk**
## ref gitlab 469

First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [ ] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [ ] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [ ] Rebase before creating a PR to keep commit history clear.
* [ ] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

---
This change is [Reviewable](https://reviewable.io/reviews/ant-design/ant-design-mobile/1603)

### CODE REVIEW

**Comment by paranoidjk** [comment]
@silentcloud rn 忘记改了... 需要同步更新吗？

**Comment by silentcloud** [comment]
@paranoidjk 加一下吧，保持一致

File: components/modal/Modal.web.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export default class Modal extends ModalComponent<ModalProps, any> {
       buttonStyle = button.style;
       if (typeof buttonStyle === 'string') {
         const styleMap = {
-          cancel: { fontWeight: 'bold' },
```
Comment (silentcloud) [inline components/modal/Modal.web.tsx]: 这里是视觉让去掉的么？cancel 这种 style 类型是和 rn 样式保持一致的 http://facebook.github.io/react-native/docs/alert.html

Comment (paranoidjk) [inline components/modal/Modal.web.tsx]: 是的，看的写的  gitlab # 469

File: components/modal/AlertContainer.tsx [hunk 2]
```diff
@@ -56,7 +63,7 @@ export default class AlertContainer extends React.Component<AlertContainerProps,
         onAnimationEnd={onAnimationEnd}
       >
         <ScrollView>
-          <Text>{content}</Text>
+          <Text style={styles.content}>{content}</Text>
```
Comment (paranoidjk) [inline components/modal/AlertContainer.tsx]: @silentcloud 改 rn 的时候给这里加 style 就显示不出来了，也没有报错，inspector 也没有这一块，去掉 style 就好了。 啥原因？

**Pull request merged**